File: level 080/classwork/codwars3.js
//https://www.codewars.com/kata/527e4141bb2ea5ea4f00072f/train/javascript
function twistedSum(n) {
  let result = 0; 
  for (let i = 1; i <= n; i++) {
    let j = i; 
    while (j > 0) {
      result += j % 10; 
      j = Math.floor(j / 10);
    }
  }
  return result; 
}

File: level 048/homework/codwars3.py
#https://www.codewars.com/kata/57eae65a4321032ce000002d/train/python
def fake_bin(x):
    result = ""
    for num in x:
        if int(num) < 5:
            result = result + "0"
        else:
            result = result + "1"
    return result